tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -254,8 +261,61 @@ describe(__filename, () => {
     expect(statsLink).toHaveProp('href', '/addon/coolio/statistics/');
   });
 
-  it('links to version history', () => {
-    const addon = createInternalAddon({ ...fakeAddon, slug: 'some-slug' });
+  // Add-ons can have a zero version! So we originally thought to check
+  // for a "0" version value (which themes will have set). But they are
+  // valid–this test is here to remind us of this :-)
+  //
+  // See:
+  // https://github.com/mozilla/addons-frontend/pull/3271#discussion_r142159199
+  it('shows verson even if it is zero', () => {
```

Comment:
I am not a fan of adding tests like this unless they are a regression test for a specific bug that actually happened. The problem is that every test adds more time to the suite so adding tests that don't prevent any known bugs do incur a cost.

If you really think we need a reminder, I would suggest adding a comment to the Flow type like this:

```diff
diff --git a/src/core/types/addons.js b/src/core/types/addons.js
index 6be02718..3a45668c 100644
--- a/src/core/types/addons.js
+++ b/src/core/types/addons.js
@@ -38,6 +38,7 @@ export type AddonVersionType = {|
   // The `text` property is omitted from addon.current_version.license.
   license: { name: string, url: string },
   reviewed: Date,
+  // This is the developer defined version number. It could, for example, be set to "0".
   version: string,
 |};
 
```

Also, it could be helpful to update [`fakeTheme`](https://github.com/mozilla/addons-frontend/blob/master/tests/unit/amo/helpers.js#L96) so that it has `version: "0"`.

Reply:
It's a bug that would have happened and I think tests make it more obvious than the type. Even if we had complete flow coverage "0" would be a valid type but one might still think to hide it as an invalid version. The tests are here to tell us how the app should work and I think a version of "0", because it isn't immediately obvious as valid, should be called out as such.